packages/react-core/src/utils/InvalidColumnError.js

```diff
@@ -0,0 +1,16 @@
+const NAME = 'InvalidColumnError';
+const MESSAGE_START = `Uncaught ${NAME}:`;
+
+export class InvalidColumnError extends Error {
+  static message =
+    'The column selected for this widget is no longer available in the data source.';
```

Comment:
Minor: i don't think message copy belongs to "react-core" and exception type, it's purely widgets & UI matter (and it's used only once there) so maybe move it there?

This should be in `[useWidgetFetch.js]` (or in UI component).

Reply:
It used in @carto/react-api also.

Reply:
Oh, now I read it correctly. I can move the message to the useWidgetFetch, ok.